test-workspace/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you accidentally copied the `test-workspace` folder from the from the `test` folder to the `root`.

Reply:
I don't see you did. Regardless, it's not quite necessary.
We're ok with a `test-workspace` inside the `test` folder, we use it for testing and we crafted our `.gitignore` to ignore just its content.
But you accidentally copied `test/test-workspace` folder to `test-workspace`. Simply delete `test-workspace` from the root (only from the root) and we're good.